conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
 - cuda-cudart-dev
 - cuda-nvcc
 - cuda-version=12.9
-- cupy>=13.6.0
+- cupy>=14.0.0
```

Comment:
Where this version upgrade comes from?
Are we comfortable of upgrading cupy?


cc: @grlee77

Reply:
Thanks! I just saw the comment (https://github.com/rapidsai/cucim/pull/978#discussion_r2586627757) :) 
Looks good to me!

Reply:
discussed offline that we should leave at 13.6 as 14.0 is still in alpha and requires a `--pre` flag to install via pip